hash/dragon_cart.xml

```diff
@@ -3,24 +3,24 @@
 <!--
 license:CC0-1.0
 
-Compiled by K1W1 and Cowering (from GoodCoCo)
+	Nearly all of the cartridges on this Dragon softwarelist will also run on the CoCo 2/3, CP400.
+	
+	ROMs dumped from RAM and marked as "bad dumps":
+		- Cumana DOS v2.0
 
-  * Nearly all of the cartridges on this Dragon softwarelist will also run on the CoCo 2/3, CP400.
+	TO DO: Most cartridges with two EPROMs have been dumped as one single file, such entries should be marked as "bad dump" until proper split dumps exist.
```

Comment:
If you write “TODO” in all caps with no space, editors and analytics tools pick it up.

The initial tabs inside the comment here will be replaced with spaces by srcclean because the opening delimiter of the comment is at the left margin.  You should probably expand them yourself to avoid churn.